changelog

```diff
@@ -1,3 +1,166 @@
+      * Ensured all ship, thumbnail, and outfit sprites have even dimensions to avoid blurriness caused by scaling. (@Anarchist2)
+      * Fixed a job with impossible to obtain offer conditions. (@Terin)
+      * Fixed a mission that was assigning reputation for a government that no longer exists. (@Amazinite)
+    * Engine bugs:
+      * Ships that are already being carried are no longer double-counted when calculating the number of available bays. (@benflodge)
+      * The destination of an invisible mission is no longer visible when opening the map during a conversation or dialog from the mission. (@Amazinite)
+      * Swizzles are now applied to carried ships when they are placed. (@quyykk)
+      * Cargo names in the ship info panel are now only capitalized after a space. (@Terin)
+      * Fixed the centering of text in the map outfitter panel. (@oo13)
+      * Fixed a longstanding bug that prevented carriers from transferring energy and fuel to carried ships. (@vitalchip)
+      * Mission NPCs no longer reparent to other NPCs of the same mission that have yet to spawn. (@Amazinite)
+      * Fixed a possible segfault caused by disowning a ship without a suitable flagship. (@petervdmeer)
+      * The hidden tag on systems can now be properly removed by events. (@Amazinite)
+  * Game content:
+    * New content:
+      * Added more spaceport news for Coalition, Hai, human, Remnant, Wanderer, and Quarg planets. (@Anarchist2, @Arachi-Lover, @BlazingDiesel, @jerith, @NRK4, @petervdmeer, @roadrunner56, @tibetiroka)
+      * Created two new civilian Coalition outfits: a fuel pod and a power generation outfit that does not rely on solar power. (@Arachi-Lover)
+      * Added more civilian hails. (@petervdmeer)
+      * The Sol system now contains all nine planets. (@ravenshining)
+      * Created a new devastating but risky weapon for Ka'het drones. (@beccabunny)
+      * More Remnant bounty hunting jobs to hunt down the new Korath Dredgers. (@arkhne)
+      * Added an additional mission after the Eye opens in the Wanderer campaign to explain to the player that they need to wait for the next part of the campaign to start. (@MasterOfGrey, @Terin)
+      * Added "spacediving" jobs to human space that can have... unpredictable outcomes. (@petervdmeer)
+      * Three new missions in Coalition space exploring Coalition folklore. (@Arachi-Lover)
+      * Created a mission in Coalition space involving running into an unlikely character. (@Arachi-Lover)
+      * Created a hardpoint sprite for the nuclear missile. (@Amazinite)
+      * Added a new person ship. (@Brick63, @Zitchas)
+      * Created reverse thruster versions for human ion and plasma engines. (@ravenshining, @Zitchas)
+      * Created lighter versions of various human secondary weapons for easier use on smaller ships. (@Pointedstick, @ravenshining, @Zitchas)
+      * Added a pirate mission involving a mafia with a penitent for extorting unwilling victims. (@petervdmeer)
```

Comment:
'penitent' doesn't make sense to me in this context. Perhaps it was meant to be 'penchant'?

Reply:
Yeah, that's a typo on my part, and a pretty egregious one at that.

Reply:
```suggestion
      * Added a pirate mission involving a mafia with a penchant for extorting unwilling victims. (@petervdmeer)
```